bin/deploy-hobby

```diff
@@ -31,7 +33,7 @@ echo ""
 echo "Do you have a Sentry DSN you would like for debugging should something go wrong?"
 echo "If you do enter it now, otherwise just hit enter to continue"
 read -r SENTRY_DSN
-export SENTRY_DSN="${SENTRY_DSN:-'https://public@sentry.example.com/1'}" $SENTRY_DSN
```

Comment:
I don't get why this were here but I might be missing something

Reply:
Why is there a sentry DSN here? Great question. Basically want to make it a bit easier to debug the stack should the user want to. This is kind of similar to what CH does https://clickhouse.com/docs/en/operations/server-configuration-parameters/settings/#server_configuration_parameters-send_crash_reports

We could reset this to an endpoint that we control by default to catch errors quicker honestly, but that seems a bit questionable from data collection POV.

Reply:
I think it's in reference to the extra `$SENTRY_DSN` rather than why it's there?

Reply:
i.e. it works but it's also going to be exporting a var with a name of whatever `$SENTRY_DSN` evaluates to

Reply:
My question was about the 2nd `$SENTRY_DSN` as I didn't get why it was needed